Berksfile

```diff
@@ -1,7 +1,9 @@
 source 'https://supermarket.chef.io'
 
 metadata
+
 group :integration do
+  cookbook 'test', path: 'test/fixtures/cookbooks/test'
   cookbook 'apt'
   cookbook 'curl'
```

Comment:
These two can get removed, they should be in the metadata of the cookbook not here :)